intents-operator/templates/intents-operator-controller-manager-servicemonitor.yaml

```diff
@@ -0,0 +1,32 @@
+{{- with .Values.serviceMonitor }}
+{{- if .enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: intents-operator-controller-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- with .Values.global.commonLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+    app.kubernetes.io/version: {{ .Chart.Version }}
+    app: intents-operator
+    release: prometheus
+  annotations:
+    {{- with .Values.global.commonAnnotations }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+    app.kubernetes.io/version: {{ .Chart.Version }}
+spec:
+  endpoints:
+  - port: https
```

Comment:
Does this work? I think the metrics are served on port 2112.
Can you create a helper template (https://github.com/otterize/helm-charts/blob/main/intents-operator/templates/_helpers.tpl) for the port, and then use the template in both places?

Reply:
It works, as we are referring to the same metrics port 2112.  The service definition for intents-operator-controller-manager-metrics-service has it is labelled as `https`, see service definition below

``` ports:
  - name: https
    port: 2112
    protocol: TCP
    targetPort: 2112
  selector:
    app: intents-operator
```

Maybe we could change the port name for service definition at some point for cleaner name.

Reply:
@seeker815 Please fix the port name in the Service then :) The port in the Deployment is correctly labeled "metrics"

Reply:
I will change that as well, thanks